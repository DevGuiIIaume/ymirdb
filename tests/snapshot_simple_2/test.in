set a 1
snapshot
set a 2
snapshot
set a 3
snapshot
set a 4
list entries
list snapshots
checkout 1
get a
set a 5
get a
checkout 2
get a
set a 5
get a
checkout 3
get a
set a 5
get a
checkout 1
get a
checkout 2
get a
checkout 3
get a
bye
