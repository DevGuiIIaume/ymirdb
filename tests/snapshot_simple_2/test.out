> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> ok

> a [4]

> 3
2
1

> ok

> [1]

> ok

> [5]

> ok

> [2]

> ok

> [5]

> ok

> [3]

> ok

> [5]

> ok

> [1]

> ok

> [2]

> ok

> [3]

> bye
